packages/react-ui-core/src/Gmap/InfoWindow.js

```diff
@@ -0,0 +1,79 @@
+import React, { PureComponent } from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import { setupEvents } from './utils/mapEventHelpers'
+
+const EVENTS = {
+  onCloseClick: 'closeclick',
+  onContentChanged: 'content_changed',
+  onDomReady: 'domready',
+  onPositionChanged: 'position_changed',
+  onZIndexChanged: 'zindex_changed',
+}
+
+const EVENT_NAMES = Object.keys(EVENTS)
+
+export default class InfoWindow extends PureComponent {
+  static propTypes = {
+    map: PropTypes.object,
+    anchor: PropTypes.object,
+    children: PropTypes.node,
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.initInfoWindow()
+  }
+
+  componentDidMount() {
+    this.infowindow.setContent(this.container)
+    this.openWindow()
```

Comment:
Do we really want to open this on mount or should do the same exact check for `anchor` we do on update?

Reply:
Good point! I'm good with it then 😄